README.md

```diff
@@ -3,16 +3,17 @@
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![NuGet Version](https://img.shields.io/nuget/v/Styra?style=flat&color=%2324b6e0)](https://www.nuget.org/packages/Styra/)
 
+> [!NOTE]
+> The canonical reference documentation is available at https://styrainc.github.io/opa-csharp/
 
-<!-- Start SDK Installation [installation] -->
 ## SDK Installation
 
 ### Nuget
 
 ```bash
-dotnet add reference path/to/Styra/OpenApi.csproj
+dotnet add reference Styra.Opa
```

Comment:
> dotnet add package Styra

This is what nuget says; I've used that for https://docs.styra.com/sdk

Reply:
This is a draft PR because I'm attempting to make it the "final state".

I've asked Philip to try to move the porcelain to Styra.Opa

And see https://github.com/StyraInc/docs/pull/85/files#diff-c24557471a377733b1b98be10ef83e910a32adb027a767dd532e2c322e71c501R70

Reply:
Sorry -- my focus here was `dotnet add reference` vs `dotnet add package` -- I know too little about dotnet to say if this is a relevant difference.